docs/book/releases/v2.1.0.md

```diff
@@ -48,6 +48,11 @@
    - Impact: In-tree vSphere volumes will not get migrated successfully
    - Workaround:
       - Upgrade CSI driver with this fix.
+7. When pod using a PVC is rescheduled to other node when metadatasyncer is down, fullsync might fail with error `Duplicated entity for each entity type in one cluster is found`.
```

Comment:
Can you add this in v2.1.1.md, just added by @RaunakShah

Reply:
Done.